File: src/main/java/com/github/bgora/Car.java
package com.github.bgora;

public record Car(String name, Integer horsePower, Integer maxSpeed) {

    @Override
    public String toString() {
        return "Car{" +
                "name='" + name + '\'' +
                ", horsePower=" + horsePower +
                ", maxSpeed=" + maxSpeed + "km/h"+
                '}';
    }
}

File: src/main/kotlin/com/github/bgora/Car.kt
package com.github.bgora

@JvmRecord
data class Car(val name: String, val horsePower: Int, val maxSpeed: Int) {

    override fun toString(): String {
        return "Car{" +
                "name='" + name + '\'' +
                ", horsePower=" + horsePower +
                ", maxSpeed=" + maxSpeed + "km/h" +
                '}'
    }
}